Common/Gfx/Canvas.cpp

```diff
@@ -563,6 +570,29 @@ void Canvas::DrawMaskedBitmap(Gdiplus::Bitmap* bitmap, Gdiplus::Bitmap* maskBitm
 	bitmapLock->Release();
 }
 
+void Canvas::DrawGeometry(const Shape & shape, D2D1_MATRIX_3X2_F & transform)
+{
+	if (!BeginTargetDraw()) return;
+
+	m_Target->SetAntialiasMode(D2D1_ANTIALIAS_MODE_ALIASED);
```

Comment:
Suddenly did the thread expand quite a bit, i'll fix it now ;)

Yeah, this would probably be better, i simply mirrored the DrawMaskedBitmap...